keps/sig-api-machinery/2161-apiserver-default-labels/kep.yaml

```diff
@@ -23,9 +24,7 @@ replaces:
 stage: alpha
 latest-milestone: "v1.21"
 milestone:
-  alpha: "v1.21"
```

Comment:
I'm surprised to see this go directly to stable instead of going through an alpha stage.  why are you skipping the standard feature lifecycle?

Reply:
the overall ask from the original kep was to skip the feature lifecycle because its non disruptive https://github.com/kubernetes/enhancements/pull/2162#discussion_r557548805 , cc @liggitt @thockin i beleive had discussed this on and off, but we eventually ended up on the side of "this is benign and so , lets just move forward with it"